package com.day10;

public class ExceptionTest01 {

	public static void main(String[] args) {
		String[] message = {"one","two","three"};
//		for(String m : message) {
//			System.out.println(m);
//		}
		
//		for(int i=0; i<message.length; i++) {
//			System.out.println(message[i]);
//		}
		
		for(int i=0; i<message.length; i++) {
			System.out.println(message[i]);
		}
	System.out.println(str.length());
		String str = null;
		
		try {
		
		}catch(NullPointerException n){
			System.out.println("Null 값"); // 예외 처리(Exception) try~catch로 사용
		}catch (ArrayIndexOutOfBoundsException a) {
			System.out.println("배열 범위 초과");
		}
		finally { // 반드시 수행되는 구문
			System.out.println("반드시 수행");
		}
		
		
	}

}
